src/azul/openapi/params.py

```diff
@@ -35,7 +47,7 @@ def path(name: str, type_: TYPE, **kwargs: PrimitiveJSON) -> JSON:
     return _make_param(name, in_='path', type_=type_, **kwargs)
 
 
-def query(name: str, type_: Union[TYPE, schema.optional], **kwargs: PrimitiveJSON) -> JSON:
+def query(name: str, type_: Union[TYPE, schema.optional], **kwargs: AnyJSON) -> JSON:
```

Comment:
Can you point me at one call site where a keyword argument is passed that is not a primitive JSON value? If there is none, this should be reverted.

Reply:
It will be reverted. This change was made when I passed `content` as a keyword arg for the `filters` parameter, and I forgot to revert it when I dropped that approach.

Reply:
> It will be reverted.

In that case, why is that change still here?

Reply:
It was reverted [here](https://github.com/DataBiosphere/azul/pull/1647/commits/523b5d36f1202ea097641faf1a6a1759421f4c74) following your refactoring how the `content` parameter is handled, and then un-reverted [here](https://github.com/DataBiosphere/azul/pull/1647/commits/681e178dbcb2ff21be92fbb825fedcb28af5717f) to accommodate the `examples` parameter.

Reply:
Sounds good.